Libraries/Text/Text/RCTTextView.m

```diff
@@ -104,15 +104,9 @@ - (NSRect)focusRingMaskBounds {
 #if DEBUG // TODO(macOS GH#774) description is a debug-only feature
 - (NSString *)description
 {
-  NSString *superDescription = super.description;
-  NSRange semicolonRange = [superDescription rangeOfString:@";"];
-  NSString *replacement = [NSString stringWithFormat:@"; reactTag: %@; text: %@", self.reactTag, _textStorage.string];
-  // [TODO(macOS GH#774): super.description isn't guaranteed to have a semicolon in it on macOS
-  if (semicolonRange.location == NSNotFound) {
-    return [superDescription stringByAppendingString:replacement];
-  } else {
-    return [superDescription stringByReplacingCharactersInRange:semicolonRange withString:replacement];
-  }
+  // [TODO(macOS GH#774): we shouldn't make assumptions on what super's description is. Just append additional content.
```

Comment:
Do we need to nil check of these properties. Does it crash when it tries to concatenate?

Reply:
`stringWithFormat` would crash if we passed in nil instead of a formatted string, but we're formatting it. You can pass in nil to a formatted string and it just replaces `%@` with `(null)`

We aren't passing in nil to `stringByAppendingString`